azext_iot/central/commands_device_group.py

```diff
@@ -24,3 +26,87 @@ def list_device_groups(
         cmd=cmd, app_id=app_id, token=token, api_version=api_version
     )
     return provider.list_device_groups(central_dns_suffix=central_dns_suffix)
+
+
+def get_device_group(
+    cmd,
+    app_id: str,
+    device_group_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> Union[DeviceGroupPreview, DeviceGroupV1_1_preview]:
+    provider = CentralDeviceGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+    return provider.get_device_group(device_group_id=device_group_id, central_dns_suffix=central_dns_suffix)
+
+
+def create_device_group(
+    cmd,
+    app_id: str,
+    device_group_id: str,
+    content: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+):
+    if not isinstance(content, str):
+        raise InvalidArgumentValueError("content must be a string: {}".format(content))
+
+    payload = utility.process_json_arg(content, argument_name="content")
+
+    provider = CentralDeviceGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    device_group = provider.create_device_group(
+        device_group_id=device_group_id,
+        payload=payload,
+        central_dns_suffix=central_dns_suffix,
+    )
+    return device_group
+
+
+def update_device_group(
+    cmd,
+    app_id: str,
+    device_group_id: str,
+    content: str,
+    central_dns_suffix: str,
```

Comment:
why is there no default argument for this central_dns_suffix?

Reply:
Actually the azure pipeline check failed this part. Where should I remove the default parameter instead here? 😲 
```
-  FAIL - HIGH severity: no_parameter_defaults_for_update_commands
    Parameter: iot central device-group update, `api_version` - Update commands should not have parameters with default values. 'api_version' in 'iot central device-group update' has a default value of '1.1-preview'
    Parameter: iot central device-group update, `central_dns_suffix` - Update commands should not have parameters with default values. 'central_dns_suffix' in 'iot central device-group update' has a default value of 'azureiotcentral.com'
```

Reply:
if you look at other commands, you will see that you want a default central_dns_suffix. To make the linter not angry like here, you will need to edit linter_exclusions.yml which is in the root directory of the this project

Reply:
Gotcha. Thanks! That's what I was trying to figure out.